lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -190,7 +185,7 @@ UiFactory<TProps> Function(UiFactory<TProps>) connect<TReduxState, TProps extend
   }
 
   UiFactory<TProps> wrapWithConnect(UiFactory<TProps> factory) {
-    final dartComponentFactory = factory().componentFactory;
+    final dartComponentFactory = factory().componentFactory!;
```

Comment:
Does this deserve a comment explaining why this is safe?

Reply:
Added here 203412867456a68a5468ea5a5f598ee535e393d0